cypress/fixtures/example.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Hva er dette?

Reply:
Den genereres av cypress om den ikke finnes. Er et område for å legge inn testdata.